### DISCUSSION TITLE
Sort plugin: Allow as item.id or item.key strings and not only integers

### DISCUSSION THREAD

**Discussion by mynona** [alpinejs/alpine#4189] [7. Bugs (archived)]
The sortable framework supports not only integers as item ids or keys. Please make Alpine also understand other identifiers.

Example: Usage of uuids as unique values for the items that should be sorted.

This would simplify the usage of the sort plugin. Thank you.

**Comment by mynona** [comment]
![Bildschirmfoto 2024-05-03 um 15 12 24](https://github.com/alpinejs/alpine/assets/4067383/f22534d2-3358-4c2e-8e90-24c5b14bb2ba)

**Reply by SimoTod** [reply]
I think you need to add quotes otherwise is not a valid string `sort:key="'YOUR STRING'"`
You may be confused by the html attribute double quotes but those are not passed into Alpine.

**Comment by mynona** [comment]
Perfect that works. Will create a pull request to update the documentation. The whole item.key part is not described.
